### DISCUSSION TITLE
export PDF failed for one file but not the if I split it

### DISCUSSION THREAD

**Discussion by vincentlc** [jgraph/drawio#2972] [Q&A]
Hello, 
I am trying to export a drawing to pdf from draw.io on google drive on google chrome.
However when I do so if I export the complete document I am facing the following issue :
![imagen](https://user-images.githubusercontent.com/16580644/182666159-5c62e8ca-0d2e-455d-9ca7-a6826db21728.png)

And it doesn't generate the pdf.
However if I do the same with this document but I only export a range of pages ( but not all), then i do success to export it to pdf.
I have also check on an other computer and web browser with the same result.
So I was wondering to what is it connected ? to the size of the document ? Or is it something else that can be fixed ?
Thanks in advance
Regards

**Comment by davidjgraph** [comment]
does it work in incognito?

**Reply by vincentlc** [reply]
I just tried and it doesn't work on incognito,  it bring the same error
